projects/manageiq.py

```diff
@@ -77,6 +77,12 @@ def deploy():
             'username: root', 'username: {0}'.format(env.user))
         run('bin/rake db:migrate')
 
+    # enables container management UI
+    with cd('manageiq/vmdb'):
+        sed('config/vmdb.tmpl.yml',
```

Comment:
You could check `containers: true` is already there so we'll avoid to add it multiple times.